src/renderer/pages/Onboarding/Vault/ManageDynamicDerivations/ManageDynamicDerivations.tsx

```diff
@@ -0,0 +1,328 @@
+        </form>
+      </div>
+
+      <div className="w-[472px] flex flex-col bg-input-background-disabled py-4 rounded-r-lg">
+        <div className="flex items-center justify-between px-5 mt-[52px] mb-6">
+          <SmallTitleText>{t('onboarding.vault.accountsTitle')}</SmallTitleText>
+
+          <Button variant="text" suffixElement={<Icon name="magic" size={16} />} onClick={fillAccountNames}>
+            {t('onboarding.vault.fillNamesButton')}
+          </Button>
+        </div>
+        <div className="flex mx-5 py-2">
+          <FootnoteText className="w-[182px] text-text-tertiary">{t('onboarding.vault.addressColumn')}</FootnoteText>
+          <FootnoteText className="text-text-tertiary">{t('onboarding.vault.nameColumn')}</FootnoteText>
+        </div>
+        <div className="overflow-y-auto pl-3 pr-3.5">
+          {accounts.map((account, index) => (
+            <div key={getAccountId(index)}>
+              <div className="flex items-center justify-between w-full gap-2">
+                <AddressWithExplorers type="short" address={account.address} explorers={RootExplorers} />
+                <div className="flex items-center">
+                  <Input
+                    disabled={inactiveAccounts[getAccountId(index)]}
+                    wrapperClass="flex w-[214px] items-center p-3 mr-9"
+                    placeholder={t('onboarding.paritySigner.accountNamePlaceholder')}
+                    value={accountNames[getAccountId(index)] || ''}
+                    onChange={(name) => updateAccountName(name, index)}
+                  />
+                </div>
+              </div>
+              <div className="flex flex-col gap-2.5">
+                {Object.entries(chainsObject).map(([chainId, { explorers }]) => {
+                  const derivedKeys = account.derivedKeys[chainId as ChainId];
+
+                  if (!derivedKeys) return;
+
+                  return (
+                    <div key={chainId}>
+                      <div className="flex items-center ml-4">
+                        <div className="bg-divider w-[2px] h-[34px] mr-4"></div>
+                        <ChainTitle fontClass="text-text-primary" chainId={chainId as ChainId} />
+                      </div>
+                      {derivedKeys.map(({ address }, derivedKeyIndex) => (
+                        <div
+                          key={getAccountId(index, chainId, derivedKeyIndex)}
+                          className="flex gap-2 justify-between items-center"
+                        >
+                          <div className="flex items-center">
+                            <div
+                              className={cnTw(
+                                'bg-divider w-[2px] h-[50px] ml-4',
+                                derivedKeyIndex === derivedKeys.length - 1 && 'h-[26px] mb-[24px]',
+                              )}
+                            ></div>
+                            <div className="bg-divider w-[8px] h-[2px]"></div>
```

Comment:
Do we need divider as a `div`? 
If we do need, let make it self closed `<div className="bg-divider w-[8px] h-[2px]"/>`